Lib/test/test_os.py

```diff
@@ -3072,13 +3072,11 @@ def check_waitpid(self, code, exitcode, callback=None):
 
     # TODO: RUSTPYTHON (AttributeError: module 'os' has no attribute 'spawnv')
```

Comment:
Is this still true?

Reply:
Well no, this comment is not valid anymore. Cleaned up (although is might not be correct PR for this but is on similar context).